android/app/src/main/java/com/httpsms/LoginActivity.kt

```diff
@@ -127,7 +127,7 @@ class LoginActivity : AppCompatActivity() {
             return
         }
 
-        if (!URLUtil.isHttpsUrl(serverUrl.text.toString().trim())) {
+        if (Settings.isProductionEnvironment(this) && !URLUtil.isHttpsUrl(serverUrl.text.toString().trim())) {

```

Comment:
I agree that we should enforce the use of HTTPS in production. However, in dev. environment I use HTTP instead of HTTPS because it's easier than setting up an HTTPS proxy to access the API.

I'll refactor this and maybe use Gradle build configs instead of Android App Settings.

Reply:
I didn't know ngrok. it's very interesting. so, I'll use that solution and disregard these changes. btw, I suggest you to add this tip to the documentation.